packages/jest-test-result/src/__tests__/formatTestResults.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import formatTestResults from '../formatTestResults';
-import {AggregatedResult} from '../types';
+import type {AggregatedResult, AssertionResult} from '../types';
```

Comment:
@SimenB It was unnecessary to add `AssertionResult`. My mistake, see: https://github.com/facebook/jest/pull/13700#discussion_r1059620028

Reply:
doesn't hurt